### PR TITLE
Clarifies how users must configure currency types in Kubecost

### PR DESCRIPTION
**Pull request by bstuder99**
## Proposed Changes

Removes telling users they can configure currency in UI, provides info on Helm flag.

### CODE REVIEW

**Comment by bstuder99** [comment]
Getting false flagged on the link checker, ignore

File: apis/apis-overview/budget-api.md [hunk 1]
```diff
@@ -171,9 +171,13 @@ When providing values for `actions`, `percentage` refers to the percentage of `s
...
+Kubecost supports configuration of the following currency types: USD, AUD, BRL, CAD, CHF, CNY, DKK, EUR, GBP, IDR, INR, JPY, NOK, PLN, and SEK. Kubecost does *not* perform any currency conversion when switching currency types; it is for display purposes, therefore you should ideally match your currency type to the type in your original cloud bill(s).
 
-Kubecost does **not** convert spending costs to other currency types; it will only change the symbol displayed in the UI next to costs. For best results, configure your currency to what matches your spend.
+Currency type can only be changed via a [`helm` upgrade to your *values.yaml*](/install-and-configure/install/helm-install-params.md), using the flag `KubecostProductConfigs.currencyCode`. For example, if you needed to convert your currency type to EUR, you would add the following to your `helm` command:
```
Comment (thomasvn) [inline apis/apis-overview/budget-api.md:176]:
```suggestion
Currency type can only be changed via a [`helm` upgrade to your *values.yaml*](/install-and-configure/install/helm-install-params.md), using the flag `.Values.kubecostProductConfigs.currencyCode`. For example, if you needed to convert your currency type to EUR, you would add the following to your `helm` command:
```

File: apis/apis-overview/budget-api.md [hunk 2]
```diff
@@ -171,9 +171,13 @@ When providing values for `actions`, `percentage` refers to the percentage of `s
...
+
+```
+--set KubecostProductConfigs.currencyCode=EUR
```
Comment (thomasvn) [inline apis/apis-overview/budget-api.md:179]:
```suggestion
--set kubecostProductConfigs.currencyCode=EUR
```

File: install-and-configure/install/first-time-user-guide.md [hunk 3]
```diff
@@ -18,6 +18,14 @@ Due to the frequency of updates from providers, it can take anywhere from 24 to
 
 Now that your base install and CSP integrations are complete, it's time to determine the accuracy against your cloud bill. Based on different methods of cost aggregation, Kubecost should assess your billing data within a 3-5% margin of error.
 
+You can also configure your currency display type. Kubecost supports the following currency types: USD, AUD, BRL, CAD, CHF, CNY, DKK, EUR, GBP, IDR, INR, JPY, NOK, PLN, and SEK. Kubecost does *not* perform any currency conversion when switching currency types; it is for display purposes, therefore you should ideally match your currency type to the type in your original cloud bill(s).
+
+Currency type can only be changed via a [`helm` upgrade to your *values.yaml*](/install-and-configure/install/helm-install-params.md), using the flag `KubecostProductConfigs.currencyCode`. For example, if you needed to convert your currency type to EUR, you would add the following to your `helm` command:
```
Comment (thomasvn) [inline install-and-configure/install/first-time-user-guide.md:23]:
```suggestion
Currency type can only be changed via a [`helm` upgrade to your *values.yaml*](/install-and-configure/install/helm-install-params.md), using the flag `.Values.kubecostProductConfigs.currencyCode`. For example, if you needed to convert your currency type to EUR, you would add the following to your `helm` command:
```

File: install-and-configure/install/first-time-user-guide.md [hunk 4]
```diff
@@ -18,6 +18,14 @@ Due to the frequency of updates from providers, it can take anywhere from 24 to
...
+
+```
+--set KubecostProductConfigs.currencyCode=EUR
```
Comment (thomasvn) [inline install-and-configure/install/first-time-user-guide.md:26]:
```suggestion
--set kubecostProductConfigs.currencyCode=EUR
```

File: troubleshooting/frequently-asked-questions.md [hunk 5]
```diff
@@ -153,3 +147,6 @@ A. To receive access to these features, you need to properly configure Workload
 
 Q: How can I configure node-exporter to use an internal cluster IP instead of the node network?\
 A: Set the following Helm value: `prometheus.nodeExporter.hostNetwork=false`.
+
+Q: How cna I configure my displayed currency type?/
```
Comment (thomasvn) [inline troubleshooting/frequently-asked-questions.md:151]:
```suggestion
Q: How can I configure my displayed currency type?/

```

Comment (bstuder99) [reply troubleshooting/frequently-asked-questions.md:151]: oops

**Review by thomasvn — APPROVED** [review]

**Pull request merged**